### PR TITLE
Document CE Customer Touchpoints

### PR DESCRIPTION
**Pull request by alexfogg**
- Add descriptions of the main ways we interact with customers
- Add links to documents and recordings for the varying touch points
- Add more complete documentation around the security review process

### CODE REVIEW

**Comment by alexfogg** [comment]
@christinelovett @emchap @amenne @justdueck @mike-r-mclaughlin Threw this together this afternoon to help out with onboarding. Open to any feedback!

File: handbook/ce/working-with-customers.md [hunk 1]
```diff
@@ -2,15 +2,146 @@
...
-A CE can glean important insights into a customers' usage via our [instance overview dashboard](https://sourcegraph.looker.com/dashboards-next/167?Unique+Server+ID=Netflix&Salesforce+Unique+ID=). It is recommended that CEs routinely monitor this to keep a pulse on customer metrics. There is also a link to this from the Salesforce Account record.
+## Customer Touchpoints
+
+A CE works with customers in a number of different ways throughout the customer journey. Below are high-level descriptions of a few of the ways that you may work with our customers, as well as links to some supporting documents and recordings.
+
+### Discovery and Demo
+
+The initial conversation(s) with a customer can vary in length and scope, but always involve discovery and demoing. This could range from an abbreviated 30 minute intro call with a smaller prospect, or multiple hour-long calls across various stakeholders and teams at an enterprise organization.
+
+Resources
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:33]:
```suggestion
#### Resources
```

Comment (emchap) [reply handbook/ce/working-with-customers.md:33]: Just to break it out, but no obligation.

File: handbook/ce/working-with-customers.md [hunk 2]
```diff
@@ -2,15 +2,146 @@
...
+- User survey planning
+- Deployment support and checklist
+- Technical configuration review
+- Trial rollout plan
+- Communication and ongoing support
+
+Resources
+
+- [Trial and Deployment Planning Template](https://docs.google.com/spreadsheets/d/1mi_540InPEs6_xmCE2gHzw6Vt9QHDx-IdGogQZN6Ezw/edit?usp=sharing)
+- [Technical Design Document Template](https://docs.google.com/document/d/19qcdFcFpqHNE6OTgO8SwdTF7FfB4AJH6Hlqeywgv6Yc/edit) (work in progress)
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:55]:
```suggestion
- [Technical Design Document Template](https://docs.google.com/document/d/19qcdFcFpqHNE6OTgO8SwdTF7FfB4AJH6Hlqeywgv6Yc/edit) (work in progress, for accounts which may deviate from our standard technical landscape)
```

File: handbook/ce/working-with-customers.md [hunk 3]
```diff
@@ -2,15 +2,146 @@
...
+- Business use cases / metrics for success
+- Technical metrics for success
+- Business value assessment planning
+- User survey planning
+- Deployment support and checklist
+- Technical configuration review
+- Trial rollout plan
+- Communication and ongoing support
+
+Resources
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:52]:
```suggestion
#### Resources
```

File: handbook/ce/working-with-customers.md [hunk 4]
```diff
@@ -2,15 +2,146 @@
...
+### License Keys
+
+- [Creating and maintaining license keys for customers](https://about.sourcegraph.com/handbook/ce/license_keys)
+- [Recording of creating a new key for demo.sourcegraph.com](https://drive.google.com/file/d/1fYsBqdzdBLd0mzAu2FJxrWznRX0k-iqr/view?usp=sharing)
+
+### Trial / POC
+
+We run the trial with a customer. This is typically about a month. It is critical that you collaborate closely with your account executive here to ensure a successful experience for the customer.
+
+Resources
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:78]:
```suggestion
#### Resources
```

File: handbook/ce/working-with-customers.md [hunk 5]
```diff
@@ -2,15 +2,146 @@
...
+- Plan a kickoff call with our main contact(s).
+
+On the kickoff call we plan our ongoing engagement with the customer. Different customers have different needs, so it's important to talk through topics such as:
+
+- How often do they want to have check-in calls?
+- What types of trainings, webinars, or Q&A sessions could their team use? When?
+- Would they like any assistance or resources to integrate Sourcegraph into their developer onboarding process?
+- Anything else we can look to help them out with? Upcoming iniaitives they could use Sourcegraph for?
+
+Resources
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:96]:
```suggestion
#### Resources
```

File: handbook/ce/working-with-customers.md [hunk 6]
```diff
@@ -2,15 +2,146 @@
...
+- [Post-Sales Kickoff Deck Template](https://docs.google.com/presentation/d/1eTis1XiS3U1M1M1a35wBw5qwTnJSMozXcz_gXJcGXhk/edit)
+- [Sample Chorus Recording](https://chorus.ai/meeting/066B000C659C4F1D83AA465576444924)
+
+### Webinars / Trainings / Q&A Sessions
+
+The webinar you will most-often run is a Sourcegraph 101/102 (basically a longer and more in-depth version of the typical demo flow). It's important to relate use cases to customer needs.
+
+We may also hold specific topic trainings or Q&A sessions, depending on customer need.
+
+Resources
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:108]:
```suggestion
#### Resources
```

File: handbook/ce/working-with-customers.md [hunk 7]
```diff
@@ -2,15 +2,146 @@
...
+### Check-in calls
+
+Check-in calls may be weekly, bi-weekly, monthly, or as-needed. Going in to a check-in call you should be familiar with:
+
+- Usage data
+- Current Sourcegraph version & upgrade info
+- New features in the latest version(s) and/or the product roadmap
+- Any recent or active questions or support issues or feature requests from the customer
+
+Resources
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:121]:
```suggestion
#### Resources
```

File: handbook/ce/working-with-customers.md [hunk 8]
```diff
@@ -2,15 +2,146 @@
...
+
+- [Chorus recording of a typical check-in call](https://chorus.ai/meeting/84885A7398C943A3AFD32327F06A3F12?)
+
+### QBRs
+
+We often hold QBRs, or Quarterly Business Reviews (sometimes referred to as an Executive Business Review), with our customers which recaps how Sourcegraph has been integrated into and creating value for their organization over the past quarter. These QBRs are typically created and presented by the CE assigned to the account, in conjunction with the account’s AE. They are typically presented in a slide deck format to executive-level leadership, and highlight usage metrics and use cases for their org. Additionally, they are used to gather feedback and comments from leadership regarding Sourcegraph.
+
+Resources
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:129]:
```suggestion
#### Resources
```

**Review by emchap — APPROVED** [review]
Left a few suggestions, mostly formatting. This is great!

**Pull request merged**